systems.csv

```diff
@@ -246,6 +248,7 @@ US,Skybike West Palm Beach,"West Palm Beach Florida, US",nextbike_wb,https://sky
 US,Sobi Long Beach,"Long Beach, NY",sobi_long_beach,http://sobilongbeach.com/,http://sobilongbeach.com/opendata/gbfs.json
 US,Spartanburg BCycle,"Spartanburg, SC",bcycle_spartanburg,https://spartanburg.bcycle.com,https://gbfs.bcycle.com/bcycle_spartanburg/gbfs.json
 US,Spin Chicago,"Chicago, IL",spin chicago,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/chicago/gbfs.json
+US,Spin Baltimore, "Baltimore, MD,spin baltimore,https://www.spin.pm/,https://web.spin.pm/api/gbfs/v1/baltimore/free_bike_status.json
```

Comment:
This one should also be pointing at the gbfs feed not the free_bike_status feed.